bin/lintsite

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Input\ArgvInput;
+
+date_default_timezone_set('UTC');
+
+set_time_limit(0);
+
+(@include_once __DIR__ . '/../vendor/autoload.php') || @include_once __DIR__ . '/../../../autoload.php';
+
+use Symfony\Component\Console\Application;
+
+$application = new Application('TwigYard', '0.1.0');
+
+$application->add(new Symfony\Component\Yaml\Command\LintCommand());
+
+$twigLintCommand = new Symfony\Bridge\Twig\Command\LintCommand();
```

Comment:
This fails with:

Fatal error: Class 'Symfony\Component\Finder\Finder' not found in /www/vendor/symfony/twig-bridge/Command/LintCommand.php on line 138